src/Apps/Home/Components/HomeEmergingPicksArtworksRail.tsx

```diff
@@ -86,6 +86,7 @@ export const HomeEmergingPicksArtworksRailFragmentContainer = createFragmentCont
         artworksConnection(
           first: 12
           marketingCollectionID: "curators-picks-emerging"
+          input: { sort: "-decayed_merch" }
```

Comment:
#minor but it might be better not to mix the form of providing some arguments at the 'root' (like `first` and `marketingCollectionID`) and some in `input`.

Those all work as expected and get collapsed down in MP resolver, but on a client might be slightly better to keep `sort` as a 'root' argument like the other two, _or_ move the other two into `input`.